test/slurm/job_conf.xml

```diff
@@ -26,7 +27,16 @@
           -->
           <param id="docker_volumes" from_environ="GALAXY_DOCKER_VOLUMES">$defaults</param>
         </destination>
+        <destination id="local" runner="local"/>
     </destinations>
+    <tools>
+        <!-- Tools can be configured to use specific destinations or handlers,
+             identified by either the "id" or "tags" attribute.  If assigned to
+             a tag, a handler or destination that matches that tag will be
+             chosen at random.
+         -->
+        <tool id="upload1" destination="local"/>
```

Comment:
I don't know if it's needed, but I can post the list of tools that don't seem to run via slurm and instead need to stay inside the docker container (our internal users seem efficient at finding them, much to my chagrin).